src/invidious.cr

```diff
@@ -249,4 +247,19 @@ Kemal.config.app_name = "Invidious"
   Kemal.config.env = "production" if !ENV.has_key?("KEMAL_ENV")
 {% end %}
 
-Kemal.run
+Kemal.run do |config|
+  if CONFIG.socket_binding
+    socket_binding = CONFIG.socket_binding.not_nil!
+    if File.exists?(socket_binding.path)
+      File.delete(socket_binding.path)
+    end
```

Comment:
This can be simplified to `File.delete?`

https://crystal-lang.org/api/1.14.0/File.html#delete%3F(path%3APath|String)%3ABool-class-method

```suggestion
File.delete?(socket_binding.path)
```